frontend/public/components/masthead-toolbar.jsx

```diff
@@ -290,6 +290,12 @@ class MastheadToolbarContents_ extends React.Component {
               });
             },
           },
+          {
+            label: t('public~Red Hat Hybrid Cloud Console'),
+            externalLink: true,
+            href: 'https://console.redhat.com',
+            image: <img src={redhatLogoImg} alt="" />,
```

Comment:
We should be adding telemetryEvent here as well like we have in OCM menu item. This will help gather the metrics.
cc: @christianvogt 

```suggestion
            image: <img src={redhatLogoImg} alt="" />,
            callback: () => {
              fireTelemetryEvent('Launcher Menu Accessed', {
                id: 'Red Hat Hybrid Cloud Console',
                name: 'Red Hat Hybrid Cloud Console',
              });
            },
```

Reply:
updated